platerplotter/platerplotter/holding_rack_manager.py

```diff
@@ -167,8 +167,8 @@ def assign_well(self, request, sample, well):
 							sample.laboratory_sample_id + " to a new rack.")
 		elif self.holding_rack.holding_rack_type == "Family":
 			matching_proband_sample_found = True
-			# all new family samples must have a matching proband
-			if sample.is_repeat == "New":
+			# all new family samples must have a matching proband except sample_delivery_mode="Family Only" samples
+			if sample.is_repeat == "New" and sample.sample_delivery_mode != "Family Only":
```

Comment:
Just trying to think through all the various combinations to see if there are any scenarios where these ANDs should actually be ORs but I think we are okay - I guess we're only needing to add an extra exclusion to when this code used to be triggered.